OpenEdXMobile/src/main/java/org/edx/mobile/task/RegisterTask.java

```diff
@@ -9,7 +9,9 @@
 
 import javax.inject.Inject;
 
+import dagger.hilt.android.AndroidEntryPoint;
 
+@AndroidEntryPoint
```

Comment:
@HamzaIsrar12 I've updated the code to use EntryPointAccesors. Also, the app is no compile-able because the of the issues marked [here](https://github.com/openedx/edx-app-android/issues/1653).

I think this issues should be solve on another PR

Reply:
Since we are initializing the LoginApi ourselves, we should remove the Injection and have the LoginAPI private. 

i.e
```
    private Bundle parameters;
    private SocialFactory.SOCIAL_SOURCE_TYPE backstoreType;
    private String accessToken;
    private LoginAPI loginAPI;
```

Please remove the unused imports too after this change.

Reply:
@HamzaIsrar12 Ready with both changes requested